examples/qwenvl/run.py

```diff
@@ -134,7 +135,7 @@ def get_model(self):
             vocab_size=vocab_size,
             num_layers=num_layers,
             gpt_attention_plugin=use_gpt_attention_plugin,
-            paged_kv_cache=paged_kv_cache,
+            kv_cache_type=KVCacheType.PAGED,
```

Comment:
`kv_cache_type` is not a constant of `KVCacheType.PAGED` and it can be got from the `build_config`.
```suggestion
kv_cache_type = config["build_config"].get("kv_cache_type", KVCacheType.CONTINUOUS)
model_config = ModelConfig(
      max_batch_size=max_batch_size,
      num_heads=num_heads,
      num_kv_heads=num_kv_heads,
      hidden_size=hidden_size,
      vocab_size=vocab_size,
      num_layers=num_layers,
      gpt_attention_plugin=use_gpt_attention_plugin,
      kv_cache_type=kv_cache_type,
      tokens_per_block=tokens_per_block,
      remove_input_padding=remove_input_padding,
      dtype=dtype,
      quant_mode=quant_mode,
      max_prompt_embedding_table_size=max_prompt_embedding_table_size,
      max_beam_width=self.num_beams,
)
```

Reply:
This PR means that in commit [https://github.com/NVIDIA/TensorRT-LLM/pull/2199](https://github.com/NVIDIA/TensorRT-LLM/pull/2199), the configuration parameter in ModelConfig has been changed from `paged_kv_cache` to `kv_cache_type`. Therefore, in recent versions of TRTLLM, we can no longer pass `paged_kv_cache` directly as before.

![image](https://github.com/user-attachments/assets/820b2953-d227-4707-9ca5-a4d0d6e130c9)